### DISCUSSION TITLE
Add option to remove "new contributors"

### DISCUSSION THREAD

**Discussion by Katsute** [community/community#6019] [Other Feature Feedback, Questions, & Ideas] [answered]
It's nice to promote first time contributors but I would like the option to disable them as it adds unnecessary information to my release notes.

![gh-release](https://user-images.githubusercontent.com/58778985/136108633-1ae9d0ba-2c94-4b13-939d-3b3d6275c1b2.PNG)

**Comment by MylesBorins** [comment, marked as answer]
Hey @Katsute. We've been debating whether or not to include this as an option. I'll bring it up with the team again to see if this is something we want to support.

**Reply by MylesBorins** [reply]
At this time we are not going to move forward with making this section optional. If folks do not want to include this content they can delete it. It is worth mentioning that a name will only appear once in this section, so thankfully you should not expect your own name to show up again
